meta_model_building/get_opt_params.py

```diff
@@ -1,40 +1,67 @@
+import numpy as np
 import os
 import pickle
 from tpot import TPOTRegressor
 from sklearn.datasets import load_boston
 from sklearn.model_selection import train_test_split
+import sys
 import time
 
+if len(sys.argv) > 1:
+    input_file_name = sys.argv[1]
+    if sys.argv[1] == 'boston':
+        housing = load_boston()
+        data, target = housing.data, housing.target
+    else:
+        try:
+            file_data = np.genfromtxt(input_file_name, delimiter=',')
+            data, target = file_data[:, :-1], file_data[:, -1:] 
+            target = np.ravel(target)
+        except:
```

Comment:
Catch the specific errors.

Also make `try` scopes as small as possible so there is less that can "go wrong".

Reply:
Fixed in dev_get-boston-data_20180924.  Also cleaned up the logic in this part of the code.